test/quantization/core/test_quantized_module.py

```diff
@@ -472,17 +508,17 @@ def test_conv1d_api(self):
                 in_channels_per_group, input_feature_map_size,
                 out_channels_per_group, groups, kernel_size, stride, pad, pad_mode,
                 dilation, X_scale, X_zero_point, W_scale, W_zero_point, Y_scale,
-                Y_zero_point, use_bias, use_fused, use_channelwise)
+                Y_zero_point, use_bias, post_op, use_channelwise)
 
     @override_qengines
     def test_conv2d_api(self):
         options = itertools.product(
             ["zeros", "reflect"],  # pad_mode
             [True, False],  # use_bias
-            [True, False],  # use_fused
+            ["none", "relu"],  # post_op
```

Comment:
nit: can be done in a separate PR, but we can split this into multiple tests as well

Reply:
Thanks for the suggestions. Split this into multiple tests and so for the other similar test cases.